.github/actions/verify-token/src/validate.ts

```diff
@@ -98,6 +98,26 @@ export function validateGitHubFields(gho: githubObj): void {
   );
 }
 
+export function validateAndMaskInputs(
```

Comment:
could you add a unit test for this? I think it should be fairly easy

Reply:
+1, will do.